backend/dal/Migrations/v01.06.03/Down/PostDown/01-Buildings.sql

```diff
@@ -0,0 +1,9 @@
+PRINT 'Update Buildings'
```

Comment:
Seems odd you're moving the SQL files associated with @LawrenceLau2020 above?

Reply:
yeah, I'll look into that.